endpoint.go

```diff
@@ -127,23 +113,33 @@ func (e *Endpoint) Addr() net.Addr {
 	return e.addr
 }
 
-func (e *Endpoint) WriteReliablePacket(buf []byte, addr net.Addr) error {
+func (e *Endpoint) WriteReliablePacket(buf []byte, addr net.Addr, opts ...ConnOption) error {
 	conn := e.getConn(addr)
 	if conn == nil {
 		return io.EOF
 	}
+
+	for _, opt := range opts {
+		opt.applyConn(conn)
+	}
+
 	return conn.WriteReliablePacket(buf)
```

Comment:
warning: exported method Endpoint.Addr should have comment or be unexported (golint)